src/home_robot/home_robot/manipulation/voxel_grasps.py

```diff
@@ -0,0 +1,211 @@
+from typing import Iterable, List, Optional, Tuple
```

Comment:
To be consistent lets add the information about all variables in the doc strings? ChatGPT might be able to help you.

Reply:
yeah this is a good idea, we should do this in general. chatgpt is the move